src/content/containers/Root.js

```diff
@@ -22,13 +22,15 @@ const LOADING_MESSAGES = Object.freeze({
   'from-file': 'Reading the file and parsing the profile in it...',
   'local': 'Not implemented yet.',
   'public': 'Retrieving profile from the public profile store...',
+  'from-url': 'Retrieving profile from URL...',
 });
 
 const ERROR_MESSAGES = Object.freeze({
   'from-addon': "Couldn't retrieve the profile from the gecko profiler addon.",
   'from-file': "Couldn't read the file or parse the profile in it.",
   'local': 'Not implemented yet.',
   'public': "Couldn't Retrieve the profile from the public profile store.",
+  'from-url': "Couldn't Retrieving profile from specified URL...",
```

Comment:
This line still has the typo. Please change it to `"Couldn't retrieve the profile from the specified URL."`.